oci-seccomp-bpf-hook.go

```diff
@@ -183,7 +192,18 @@ func startFloatingProcess() error {
 		return err
 	}
 	pid := s.Pid
-	fileName := s.Annotations["io.containers.trace-syscall"]
+	var outputFile string
+	var inputFile string
+	annotation := s.Annotations["io.containers.trace-syscall"]
+	annotationSplit := strings.SplitN(annotation, ";", 2)
+	for _, paths := range annotationSplit {
+		if strings.HasPrefix(paths, "if:") {
+			inputFile = strings.TrimSpace(strings.TrimPrefix(paths, "if:"))
+		}
+		if strings.HasPrefix(paths, "of:") {
+			outputFile = strings.TrimSpace(strings.TrimPrefix(paths, "of:"))
+		}
+	}
```

Comment:
There should be a log.Error here. Stating that the anotation was not an input or an output file.